tascrum/serializers.py

```diff
@@ -273,9 +296,14 @@ class Meta:
         fields = ['id','order','title','list','members','role','labels','startdate','duedate','reminder', 'storypoint', 'setestimate','description','status','comment']
 
     def get_role(self, obj):
-        roles = obj.crole.all()
+        roles = obj.crole.exclude(role='owner')
         return CardRoleSerializer(roles, many=True).data
     
+    # def get_members(self, obj):
+
+        # members = Member.objects.filter(cmembers__card=obj, cmembers__role__ne='owner')
+        # return CardMemberSerializer(members, many=True).data
```

Comment:
why did you comment this? 
owner needs to be checked.

Reply:
it causes errors